### PR TITLE
Make README a little better

### PR DESCRIPTION
**Pull request by devxan**
So your README file was unprofessional and not very user-friendly. This pull request makes it all better. (A little bit.)

### CODE REVIEW

**Review by devxan — COMMENTED** [review]
With six commits, this pull request will bring the README of the public MoDo repository to the **next level**.

**Comment by DaRealDorseyBro** [comment]
> So your README file was unprofessional and not very user-friendly. This pull request makes it all better. (A little bit.)

Hey Xan! Didn't expect you to find me on GitHub haha, yea my README is pretty unprofessional since I just copied it from an old description I made, I plan to update it and also [_check out my site_](https://roseys-roses.wtf)

File: README.md [hunk 1]
```diff
@@ -1,16 +1,17 @@
 # MoDo, the last bot you'll need
 ### What is MoDo?
-MoDo is a simple discord bot that has many features and commands, each command is simple to use and is mainly self-explanatory.
+MoDo is a simple discord bot that has many features and commands, each command is simple to use and is mainly self-explanatory. The bot can be added to your server on [Top.gg](https://top.gg/bot/711421867776278579) or with the Invite button at the bottom of this README.
```
Comment (DaRealDorseyBro) [inline README.md:3]: Good idea, forgot to add it.

File: README.md [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
- - Other
+ - [Moderation](#1-moderation)
+ - [Utility](#2-utility)
+ - [Fun](#3-fun)
+ - [Music (Rate Limited)](#4-music-rate-limited)
+ - [Economy](#5-economy)
+ - [Phone](#6-phone)
+ - [Configuration](#7-configurations)
+ - [Other](#8-other)
+ - [Help](#if-you-are-confused-about-a-command)
```
Comment (DaRealDorseyBro) [inline README.md:14]: Didn't even think of this, thanks Xan!

File: README.md [hunk 3]
```diff
@@ -30,24 +31,28 @@ MoDo is a simple discord bot that has many features and commands, each command i
...
 * **Phone calls**,
-You can now call other servers! Just set the phone channel and do `m-phonebook` to see the numbers!
+You can now call other servers! Just set the phone channel and do `m-phonebook` to see the numbers.
 * **Economy**,
-MoDo Also now has a economy! Do you're daily chores and go work hard to earn some money!
+MoDo also now has economy features! Do your daily "chores" and go work hard to earn some money within your Discord server!
 * **Leveling**,
-MoDo has just came out with a leveling system! Its pretty simple, level up by being active and sending messages!
-## Thanks for looking at my bot, and remember, it only takes 6 clicks to invite him!
+MoDo has recently been given a leveling system! Its pretty simple: level up by being active and get sent congratulation messages!
```
Comment (DaRealDorseyBro) [inline README.md:54]: Thanks for all this Xan!!!!

**Review by DaRealDorseyBro — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (devxan) [reply README.md:14]: You're welcome!

File: README.md [hunk 4]
```diff
@@ -31,27 +31,28 @@ MoDo is a simple discord bot that has many features and commands, each command i
...
+If this is set to **on** it will detect and delete any discord invites!
 * **Phone calls**,
-You can now call other servers! Just set the phone channel and do `m-phonebook` to see the numbers!
+You can now call other servers! Just set the phone channel and do `m-phonebook` to see the numbers.
 * **Economy**,
-MoDo Also now has a economy! Do you're daily chores and go work hard to earn some money!
+MoDo also now has economy features! Do your daily "chores" and go work hard to earn some money within your Discord server!
 * **Leveling**,
-MoDo has just came out with a leveling system! Its pretty simple, level up by being active and sending messages!
+MoDo has recently been given a leveling system! Its pretty simple: level up by being active and get sent congratulation messages!
```
Comment (splatterxl) [inline README.md:54]: I think a semi-colon would be better here.

Comment (devxan) [reply README.md:54]: Good idea. @DaRealDorseyBro, you should merge this.

**Pull request merged**